subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -58,7 +58,7 @@ static void win_offset_calc(struct ll_conn *conn_curr, uint8_t is_select,
 #endif
 #endif /* CONFIG_BT_CTLR_CONN_PARAM_REQ */
 
-#if defined(CONFIG_BT_CONN)
+#if defined(CONFIG_BT_CONN) && (defined(CONFIG_BT_CENTRAL) || defined(CONFIG_BT_LEGACY))
```

Comment:
Since `CONFIG_BT_CENTRAL` implies `CONFIG_BT_CONN`, this should perhaps have been
```suggestion
#if (defined(CONFIG_BT_CONN) && defined(CONFIG_BT_LEGACY)) || defined(CONFIG_BT_CENTRAL)
```

Reply:
Normally yes, but I'll leave it as is, since this change has to be reverted once issue #44339 is resolved